client/src/modules/dashboard/index.jsx

```diff
@@ -6,15 +6,18 @@ import {
   ClipboardSolid as ShoppingListIcon,
   UsersSolid as CohortIcon
 } from 'assets/images/icons';
-import { fetchAllLists } from './model/actions';
-import { getCohorts, getShoppingLists } from './model/selectors';
+import { fetchShoppingLists } from 'modules/shopping-list/model/actions';
+import { fetchAllCohorts } from 'modules/cohort/model/actions';
```

Comment:
fetchShoppingLists (1) but fetchAllCohorts (2) maybe add (1) or remove (2) 'All' - that the names would be more consistent?